proto/regen/ecocredit/marketplace/v1/state.proto

```diff
@@ -125,12 +125,12 @@ message BuyOrder {
 message AllowedDenom {
   option (cosmos.orm.v1alpha1.table) = {
     id : 3
-    primary_key : {fields : "bank_denom"}
+    primary_key : {fields : "base_currency"}
     index : {id : 1 fields : "display_denom", unique : true}
   };
 
   // denom is the bank denom to allow (ex. ibc/GLKHDSG423SGS)
```

Comment:
```suggestion
  // base_currency is the bank denom to allow (ex. ibc/GLKHDSG423SGS)
```

Reply:
The description should probably be updated as well but I'm not sure how to word it because I think "bank denom" is an accurate description.